config/routes/workflow_executions.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+scope :workflow_executions, module: :workflow_executions, as: :workflow_executions do
+  resources :submission, only: %i[new show] do
```

Comment:
Yeah as per above if we are using `resources` here this should be renamed to `submissions`. As per comment on renaming controller above.

Reply:
Updated in 689a4b924d24f9c33edc059cde63b94e1b5241ea